src/util/std_code.h

```diff
@@ -302,8 +317,7 @@ inline code_declt &to_code_decl(codet &code)
   return static_cast<code_declt &>(code);
 }
 
-/*! \brief A removal of a local variable
-*/
+/// A \ref codet representing the removal of a local variable.
```

Comment:
Removal ... from where? (I know, that's what the original docs said.) Suggestion: `A \ref codet representing the effect of a local variable going out of scope.`

Reply:
I kept "removal" as it's more descriptive than "effect" but added the "going out of scope part", thanks! The reason why I didn't add this before was that I couldn't tell just from looking at the class what exactly it would be used for. :slightly_smiling_face: